### DISCUSSION TITLE
Timer for chasing spikes in Muspah

### DISCUSSION THREAD

**Discussion by Stickaplex** [runelite/runelite#16221] [Q&A] [answered]
havent seen it yet but a timer for muspahs chasing spikes (the ones that follow you around the room) Not sure if an info box would be better or an overlay on the spike itself. would make the boss a bit nicer imo

**Comment by Enriath** [comment, marked as answer]
Banned by the [Jagex Statement](https://secure.runescape.com/m=news/third-party-client-guidelines?oldschool=1)

> Indicates the time where a boss mechanic may start or end
